lib/pages/route/route_detail.dart

```diff
@@ -33,13 +34,15 @@ class _DetalleRutaScreenState extends State<DetalleRutaScreen> {
   List<dynamic>? usuariosFiltrados;
   TextEditingController _searchController = TextEditingController();
   String? errorMessage;
+  double _rating = 0.0;
 
   @override
   void initState() {
     super.initState();
     _nombreController = TextEditingController(text: widget.ruta['nombre']);
     _descripcionController = TextEditingController(text: widget.ruta['descripcion']);
     _mapController = MapController();
+    _rating = (widget.ruta['puntaje'] as num?)?.toDouble() ?? 0.0; // Asigna un puntaje inicial válido
```

Comment:
que sea por defecto null en vez de 0.0. Considerar este cambio en los otros `_rating`. Si es `null`, que se muestre como `'---'`.
```suggestion
    double? _rating = (widget.ruta['puntaje'] as num?)?.toDouble();
```